.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
```

Comment:
wouldn't we want this to be triggered upon a PR into the 'master' branch from the 'develop' branch (like in the node repo https://github.com/DataHighway-DHX/node/blob/master/.github/workflows/rust.yml#L3)?